### DISCUSSION TITLE
Prometheus Operator support for prometheus servers not deployed by Operator

### DISCUSSION THREAD

**Discussion by hemant659** [prometheus-operator/prometheus-operator#4484] [Q&A]
Can Prometheus Operator manage the Prometheus servers and other components which are not deployed by Prometheus Operator CRDs? I have existing Prometheus setup which is not deployed through CRDs introduced by Prometheus Operator. I want to add rules(through PrometheusRule CRD) and monitor targets(through ServiceMonitor CRD) on my existing Prometheus setup(not deployed by Prometheus Operator). Is this possible?
For eg, if I install Prometheus server from official website   https://prometheus.io/docs/prometheus/latest/getting_started/
Is there a way Prometheus Operator can mange these Prometheus setup(servers and other components e.g. Alert Manager)

**Comment by paulfantom** [comment]
No, such setup is not possible.

**Reply by hemant659** [reply]
Thanks Pawel
